Automattic/wp-calypso
# Packages: Build individually

Implementing https://github.com/Automattic/wp-calypso/pull/30933#issuecomment-467610569

#### Changes proposed in this Pull Request

* Replace `bin/build-packages.js` with `bin/build-package.js`, and use it from packages' individual `build` scripts.
* Invoke those from the root `package.json` through `lerna`. 

#### Testing instructions

`npm run build-packages`

should produces some messages ending in

```
lerna success run Ran npm script 'build' in 4 packages in 1.5s:
lerna success - @automattic/babel-plugin-i18n-calypso
lerna success - i18n-calypso
lerna success - photon
lerna success - @automattic/tree-select
```

Check packages' `dist/` dirs to verify that they have been built.

Furthermore, verify that there are no more references to `bin/build-packages` (plural!) in the codebase. (We reference `npm run build-packages` everywhere, so e.g. CircleCI doesn't need to be updated.)
